apps/state/lib/state/metadata.ex

```diff
@@ -46,10 +46,10 @@ defmodule State.Metadata do
   end
 
   @doc """
-  Sets latest version of the Feed.
+  Sets latest version, start_date and end_date of the Feed.
   """
-  def feed_updated(version) do
-    :ets.insert(table_name(), {State.Feed, version})
+  def feed_updated(version, start_date, end_date) do
```

Comment:
If you pass the tuple in here, you don't need to do the unwrapping in `feed_metadata/0`.